.travis.yml

```diff
@@ -2,6 +2,8 @@
 # RVM/bundler/ruby and whatnot. Right now 'rust' as a language actually
 # downloads a rust/cargo snapshot, which we don't really want for building rust.
 language: c
+compiler: /usr/bin/gcc-4.7
```

Comment:
Because we have to hardcode the path to the specific version (travis doesn't support updating default versions, and it requires sudo to do manually), neither C nor C++ will give us the desired behaviour. That is, C only sets CC and C++ does:

> For each row, the Travis CI C++ builder will export the CXX env variable to point to either g++ or clang++ and CC to either gcc or clang.